tools/make-dist

```diff
@@ -54,6 +57,7 @@ else
     ../../configure \
         CPPFLAGS=-I../../tools/mock-build-env \
         PKG_CONFIG_PATH=../../tools/mock-build-env \
+        XGETTEXT='/bin/true' \
```

Comment:
Hmm.. I *think* we use xgettext and msg{cat,merge} etc. only for building/updating po/pot files (i.e. not relevant for the tarball), but this also controls `xsltproc` -- and that we *do* need to build the tarball. This has the potential to release tarballs with broken documentation. I assume the build wouldn't fail later on, as otherwise what's the point of stubbing this out.

Reply:
Now that I think about it I actually have no idea what the deal with this `xgettext`/`xsltproc` confusion is.  There two things have absolutely nothing to do with each other other than starting with `x`.  It smells like a copy/paste error.  I'll sort that out.

In any case:
 - xgettext is only used when building a `.pot` file, but since there is a rule for this in the `Makefile`, one might assume that this rule will function properly after `./configure` as run.  If we want to change that assumption, I'd add a `--enable-pot-rules` or so, but this seems like some serious overkill.  tl;dr: Status quo is OK here I think.
 - `xsltproc` is only used for building docs and our dependency on it ought to be disabled via `--disable-doc`.
 - `msgfmt` is how we get `.mo` files for `gettext()` in C, and this is part of the main build process.  Because we like pain, we also use `msggrep` as part of that process, in a very bizarre way that I'd like to burn down.  In short: this needs to be here, until we decide to kill it.

All in all, I think everything is right for this PR, but we certainly have a lot of work we could do in a separate PR.  Let's discuss that in the meeting we have in 6 minutes :)